posthog/temporal/usage_reports/run_usage_reports.py

```diff
@@ -78,15 +81,37 @@ async def query_usage_reports(
         period = get_previous_day(at=at_date)
         period_start, period_end = period
 
+        logger.info("Querying all org reports", period_start=period_start, period_end=period_end)
+
         @sync_to_async
-        def get_all_org_reports(ps, pe):
-            return _get_all_org_reports(ps, pe)
+        def async_get_all_usage_data_as_team_rows(p_start, p_end):
+            return _get_all_usage_data_as_team_rows(p_start, p_end)
```

Comment:
Should this not also use `@database_sync_to_async`?

Reply:
I'll look before merging.